src/Strings.ts

```diff
@@ -27,6 +28,7 @@ const localizedStrings: LocalizedStrings = {
         messageFailed: "couldn't send",
         messageSending: "sending",
         timeSent: " at %1",
+        placeholder: "Type your message..."
```

Comment:
perhaps we can name this more specifically - I can see a case where there may be other "placeholders"